BrowserKit/Sources/WebEngine/WKWebview/WKEngineWebView.swift

```diff
@@ -120,11 +121,20 @@ extension WKEngineWebView {
     }
 }
 
-final class DefaultWKEngineWebView: WKWebView, WKEngineWebView, MenuHelperWebViewInterface, ThemeApplicable {
+// TODO: FXIOS-7897 #17642 Handle WKEngineWebView AccessoryViewProvider
+final class DefaultWKEngineWebView: WKWebView,
```

Comment:
Can we remove this TODO while we're here? Turns out this task was done https://mozilla-hub.atlassian.net/browse/FXIOS-7897
```suggestion
final class DefaultWKEngineWebView: WKWebView,
```